scripts/deprecate-sh.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+
+const paths = [
+    "core/main",
+    "core/editor",
+    "components/angular",
+    "components/inferno",
+    "components/jquery",
+    "components/preact",
+    "components/react",
+    "components/svelte",
+    "components/vue",
+    "components/vue3",
+    "presets/60fps",
+    "presets/backgroundMask",
+    "presets/basic",
+    "presets/bigCircles",
+    "presets/bouncing",
+    "presets/fire",
+    "presets/fontAwesome",
+    "presets/fountain",
+    "presets/snow",
+    "presets/stars",
+    "shapes/bubble",
+    "shapes/heart",
+    "shapes/multiline-text",
+    "shapes/spiral"
+];
+
+const lines = [];
+
+for (const path of paths) {
+    try {
+        const packageInfo = require(`../${path}/package.json`);
```

Comment:
Codacy found an issue: [Found non-literal argument in require](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)